connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourcesExtension.java

```diff
@@ -155,7 +155,7 @@
     public static final String SUBSYSTEM_NAME = Constants.DATASOURCES;
     private static final String RESOURCE_NAME = DataSourcesExtension.class.getPackage().getName() + ".LocalDescriptions";
 
-    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(4, 0, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(4, 0, 1);
```

Comment:
We don't do micro version bumps in community, reserving them for emergencies in EAP CPs. So 4.1.0.